pages/Announcements/Overview.md

```diff
@@ -1,101 +0,0 @@
----
```

Comment:
This content was moved to a few different places:
- pages/DSNP/Announcements.md
- pages/DSNP/Serializations.md
- pages/Ethereum/Validation.md